name.dependencies.v1.jschema

```diff
@@ -73,6 +73,7 @@
                                 "RabbitMq",
                                 "MongoDb",
                                 "SqlServer"
+								"Elasticsearch"
```

Comment:
Please maintain consistency and use spaces instead of tabs. After that i will merge